modules/steam.js

```diff
@@ -0,0 +1,119 @@
+var keys    = require('../util/keys');
+var SteamApi = require('steam-api');
+var request = require('request');;
+
+var app = new SteamApi.App(keys.STEAM.KEY);
+var player = new SteamApi.Player(keys.STEAM.KEY);
+var user = new SteamApi.User(keys.STEAM.KEY);
+
+//Returns a list of all the AppIDs and names of all the games in the steam library
+//Use this command at your own risk
+module.exports.GetGameIDs = function(message){
+  app.GetAppList().done(function(result){
+    gameList = [];
+    result.forEach(function(game){
+      gameList.push(game['name']); //gameList.push((game['appid'], game['name']));
+    });
+    gamesArr = splitStrByTextLimit(gameList.join('\n'));
+    message.channel.send("Check your DMs");
+    message.author.sendMessage("Entire Steam Library has to offer:\n");
+    gamesArr.forEach(function(block){
```

Comment:
Can we pre-make this string before sending it, then chunk that? This sounds like ping hell.

Reply:
In discord you mentioned this is a joke function. I'm not inclined to let this command in, as it may flag the attention of discord itself and get the bot banned automatically.